### DISCUSSION TITLE
Unable to apply for GitHub Global Campus despite having school issued email.

### DISCUSSION THREAD

**Discussion by woozzzy** [community/community#47719] [GitHub Education]
I am getting the error message: "We require applicants to use a school-issued email address to apply." However, I have added my school-issued email and am selecting it, but it will not let me send my application. 

My school domain is gatech.edu, and I'm selecting the "College of Computing - Georgia Tech" as my school. I'm wondering if maybe the domain is configured incorrectly. 

I am an undergraduate student, and my email is in the format **********@gatech.edu.

**Comment by jge162** [comment]
Sounds like an easy fix, just reach our to [GitHub education support.](https://support.github.com/contact/education)

**Comment by carlajarchuleta1-a11y** [comment]
******@gatech.edu.

**Reply by carlajarchuleta1-a11y** [reply]
> ******@gatech.edu.

``

**Comment by SALbestMAN** [comment]
If your university appears in the list, the issue is usually one of the following:
- You haven’t added your school email to your GitHub account
- You added it but didn’t verify it
- Your Billing information is incomplete (you don’t need to add a payment method or pay anything, but the billing details must be filled)

If your university does not appear in the list, then you need to contact GitHub Education Support so they can review your case.
